src/ray/core_worker/core_worker.cc

```diff
@@ -609,12 +609,25 @@ CoreWorker::CoreWorker(const CoreWorkerOptions &options, const WorkerID &worker_
         new raylet::RayletClient(std::move(grpc_client)));
   };
 
+  auto on_excess_queueing = [this](const ActorID &actor_id, int64_t num_queued) {
+    auto timestamp = std::chrono::duration_cast<std::chrono::seconds>(
+                         std::chrono::system_clock::now().time_since_epoch())
+                         .count();
+    std::ostringstream stream;
+    stream << "Warning: More than " << num_queued
```

Comment:
Maybe we'd like to show more metadata? Like actor name / repr (I think it is not available now) / ip address?

Reply:
I think at least ip address will be helpful

Reply:
Yeah, I have a TODO above (I think we should address this commonly as part of https://github.com/ray-project/ray/issues/15933